### DISCUSSION TITLE
Request the following functionality for the mobile version

### DISCUSSION THREAD

**Discussion by 873578156** [community/community#72738] [Mobile]
Request the following functionality for the mobile version 

0. Create files and directories. 
1. Delete files and directories. 
2. Rename files and directories. 
3. Edit file function: 
    3.1. Change the encoding of the file (ASCII, GBK, Unicode, UTF-8). 
    3.2. Changes file a newline (Windows: CRLF, Linux: LF, MAC: CR).

**Comment by mattruggio** [comment]
Hey @873578156, thanks for the feature request! I need to break this down into smaller requests:

1. Create files: this functionality should currently exist (see attachment below)
2. Create directory: this functionality should also exist (new files should have their paths properly split with forward slashes representing parent directories. For example: creating a new file with the name `1/2/3.txt` should also create directories `1` and `2` if they do not exist.)
3. Delete files: [this](https://github.com/orgs/community/discussions/71119) functionality request has recently been submitted
4. Delete directories: new request
5. Change file encoding: recommend separate community submission under the [Repository category](https://github.com/orgs/community/discussions/categories/repositories).
6. Change file new line character: recommend separate community submission under the [Repository category](https://github.com/orgs/community/discussions/categories/repositories).

I will create a new internal feature request for the directory deletion since that functionality currently exists in the web experience but not mobile. However the two file edit functions would most likely need to be submitted to a different category since this functionality does not exist yet in our web experience. Would you mind creating a new discussion in potentially [the Repository community category](https://github.com/orgs/community/discussions/categories/repositories) for those two?

![image](https://github.com/community/community/assets/906763/dea3c632-c5ad-4917-8d60-48784a18359d)

**Reply by 873578156** [reply]
Very willing.Created.
